Makefile

```diff
@@ -461,7 +468,7 @@ ci-unit-test:
 
 kill-db-container:
 	docker kill postgres
- 
+
```

Comment:
nit (probably change only if you updated the PR for other reasons): additional space.

Reply:
disregard.